### ISSUE TITLE
Add more configurability to the "Focus Highlight" function in NVDA

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10892]
### Is your feature request related to a problem? Please describe.
Depending on the background color of the application or web page in which we currently are, the currently chosen colors for the NVDA's focus highlighting may be inadequate. There may be cases in which the colors of the background and the highlighter are identical or closely identical to each other. In such cases, the focus highlighting instead of being useful, may become useless.

### Describe the solution you'd like
Add the ability to customize the parameters of the focus highlighters: the color of each highlighter, its style (e.g. thickness, solid, dash, dot, dash dot, dash dot dot, etc.). In summary, it should be something like the "Focus Highlight" add-on in this regard.

**Comment by feerrenrut** [comment]
This would be a good improvement, probably the only reason this hasn't been done so far is because some thought needs to be put into for who and why these parameters need configuration. That is, defining the use-cases. Often the answer to 'why do you want to configure this'  tells us a lot about what an acceptable solution would be. For instance: "I want to be able to change the color of the focus highlight rectangle so that the contrast with the background is higher", then perhaps another solution is to draw an outline for the focus highlight rectangle in a contrasting color. 

To move this forward it would be helpful to enumerate the parameters you would like to be able to configure and a very brief reason why it is helpful.

**Comment by k-kolev1985** [comment]
Most important for me is to be able to configure the color for each highlighter. For example, under normal circumstances, red is not so bad for the object navigator, but I use the Windows high-contrast black theme in which the background is black. In this case, a brighter color (e.g. bright green) would be more suitable.

But not all users of the focus highlight feature use a high-contrast black theme (they use a normal theme with a white background), so the bright green color preferred by me will probably not be suitable for them. Each user may have different color preferences. That is why I proposed the configurability - so each user can customize the highlighter to their own preferences.

As for the shape and form of the highlighter. I don't mind the current configuration. But I proposed the configurability, because some users may prefer to change this aspect of the highlighter.
